src/lolex-src.js

```diff
@@ -568,6 +570,13 @@ function withGlobal(_global) {
                 delay: timeout
             });
         };
+        if (symbolUtilPromisifyCustom) {
+            clock.setTimeout[symbolUtilPromisifyCustom] = function (delay) {
+                return new _global.Promise(function (resolve) {
```

Comment:
Hmm, can you elaborate on that?

Reply:
People have tests that do

```js
global.Promise = undefined
```

inside of their test. It would be great to be resilient to that by grabbing a copy of the original `Promise` early

Reply:
@SimenB You're basically just saying that `lolex` should do something like this, right?
```javascript
// needed not to break in case people null out the `Promise` property in user-code
var originalPromise = _global.Promise;
```

And then reference that when using `Promise`s internally.

Reply:
yes 🙂